samples/tfm/tfm_psa_template/sysbuild/mcuboot/boards/nrf5340dk_nrf5340_cpuapp.conf

```diff
@@ -0,0 +1,23 @@
+#
+# Copyright (c) 2024 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+CONFIG_BOOT_MAX_IMG_SECTORS=256
+
+CONFIG_PCD_APP=y
```

Comment:
this entire file can go, the only thing that might be needed is the max img sectors, which can go in the mcuboot prj.conf file instead

Reply:
It was quite difficult to get the update of the netcore image to work correctly. The reason was that most of these Kconfigs were hard to get into correct position. There is no way this will work without these Kconfigs.

Reply:
There are set by sysbuild automatically

Reply:
Thanks for pointing out. This change had gone unnoticed by me.